andela/Titan-Devs
# #164238428 Increase Test Coverage

#### What does this PR do?
Increasing test coverage 
#### Description of Task to be completed?
The develop branch should have test coverage which is above 80% on statements, functions, and branches. 
#### How should this be manually tested?
* Fetch the branch `ch-increase-coverage-164238428`
* Checkout the branch `ch-increase-coverage-164238428`
* Setup your database. 
* Run tests `yarn test`
* You should see coverage which is above 80% in all aspects. 
#### What are the relevant pivotal tracker stories?
164238428
